### PR TITLE
[FSSDK-9106] chore: prepare for 5.0.0-beta

### PR DESCRIPTION
**Pull request by andrewleap-optimizely**
## Summary
- Prepare for advanced audience targeting release

## Issues
[FSSDK-9106](https://jira.sso.episerver.net/browse/FSSDK-9106)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+([#303](https://github.com/optimizely/ruby-sdk/pull/303),
+[#306](https://github.com/optimizely/ruby-sdk/pull/306),
+[#308](https://github.com/optimizely/ruby-sdk/pull/308),
+[#310](https://github.com/optimizely/ruby-sdk/pull/310),
+[#311](https://github.com/optimizely/ruby-sdk/pull/311),
+[#312](https://github.com/optimizely/ruby-sdk/pull/312),
+[#314](https://github.com/optimizely/ruby-sdk/pull/314),
+[#316](https://github.com/optimizely/ruby-sdk/pull/316),
+[#323](https://github.com/optimizely/ruby-sdk/pull/323),
+[#324](https://github.com/optimizely/ruby-sdk/pull/324)).
```
Comment (Mat001) [inline CHANGELOG.md:18]: @jaeopt was telling me to limit the number of PRs to 4-5 main ones. I wonder if that applies to here as well?

**Review by Mat001 — CHANGES_REQUESTED** [review]
Check w Jae about the number of PRs?

**Review by Mat001 — APPROVED** [review]
Correcting to "approve" status. Just check that Jae is ok with the number of PRs.

**Review by russell-loube-optimizely — COMMENTED** [review]
lgtm!

**Comment by andrewleap-optimizely** [comment]
> Correcting to "approve" status. Just check that Jae is ok with the number of PRs.

I removed a bunch

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**